packages/shared/routes/setup/Legal.svelte

```diff
@@ -609,19 +609,32 @@
                     <Text type="p" secondary classes="mb-4">
                         We use CoinGeckoAPI of Gecko Labs Pte. Ltd, 101 Upper Cross Street, #05-16 People's Park Centre, Singapore
                         058357 ("CoinGecko") to obtain historical price and market data for cryptocurrencies. Further information
-                        regarding their privacy policy can be found under https://www.coingecko.com/en/privacy
+                        regarding their privacy policy can be found under https://www.coingecko.com/en/privacy.
                     </Text>
                     <Text type="h3" classes="mb-5">GitHub</Text>
                     <Text type="p" secondary classes="mb-4">
-                        We use GitHub of GitHub Inc 88 Colin P. Kelly Street, San Francisco, CA 94107, USA ("GitHub" - in the
+                        We use GitHub of GitHub Inc., 88 Colin P. Kelly Street, San Francisco, CA 94107, USA ("GitHub" - in the
```

Comment:
```suggestion
                        We use GitHub of GitHub Inc., 88 Colin P. Kelly Street, San Francisco, CA 94107, USA (in the
```

Reply:
Ah good catch, I forgot about that change. Thanks!